src/components/ui/reusable/Link.tsx

```diff
@@ -21,9 +22,11 @@ const linkStyles = tv({
 type Props = LinkProps & VariantProps<typeof linkStyles>;
 
 export const Link: FC<PropsWithChildren<Props>> = ({ children, ...props }) => {
+	const { variant, ...restProps } = props;
```

Comment:
bych radši použil props.variant.. potřebuješ to jen na jednom místě, kdyby jsi to potřeboval na víc místech, tak by mi to potom dávalo smysl